handlers/webhook.go

```diff
@@ -0,0 +1,72 @@
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/stretchr/testify/assert"
+)
+
+type WebhookReceiver struct {
+	handler  func(w http.ResponseWriter, req *http.Request, _ httprouter.Params)
+	port     int
+	requests chan []byte
+	router   *httprouter.Router
+	server   *http.Server
+}
+
+func (s *WebhookReceiver) Init() {
```

Comment:
The webhook exists in Studio and we call out to it. I don't think we need this (but might be missing something)

Reply:
Used just for callback test. Last point in #2552 spec. Do you want it removed?

Reply:
Fixed